### ISSUE TITLE
Bump pillow version for Python 3.9 support on Windows

### ISSUE THREAD

**Issue by hofst** [SerpentAI/D3DShot#44]
Currently, d3dshot won't install with Python 3.9 on Windows.
You can fix this, by bumping the pillow version requirement to 8.2.0

**Comment by PlanetExpressShip** [comment]
Do you know how to fix the issue

**Comment by Hellikandra** [comment]
Have you tested to change the pyproject.toml =>
pillow = "~7.1.2".

You need to be sure that every version of  other package can be used by the Py 3.9.

**Comment by DanPJones** [comment]
How do you "bump the pillow version requirement to 8.2.0"?

**Comment by mathisHeidrich** [comment]
I just had to download the whole code, then unzip it and then change the line pillow = "~7.1.2" to pillow = ">=7.1.2" in the pyproject.toml. The last thing you have to do is to run pip install ./D3DShot-dev in the directory you unziped it. Just like @Hellikandra said.

**Comment by Avasam** [comment]
This has been stagnating for over a year. But if you'd like to update it yourself, create a fork, or use one of the 3 already updated forks and use `pip install git+https://github.com/<user>/D3DShot.git#egg=D3DShot`

**Comment by fauskanger** [comment]
I have made fork here with the changes mentioned above by @Tomatie

Fork: https://github.com/fauskanger/D3DShot

I cloned/copied it to my local drive and then did `pip install .` from inside the cloned repo, and it installed on Win 10, Python 3.10.4.

Does anyone have any good ideas to how I could/should distribute it when it's part of my project? Is it OK to make a new package and upload it to pypi even though it's a fork? I just want to have a way to distribute my package (that used D3DShot), but I'd like to proceed respectfully and transparently, and don't want to take credit for the work.

Is that what you suggest @avasam in your post when you say use `pip install git+https://github.com/<user>/D3DShot.git#egg=D3DShot` ?

**Comment by Avasam** [comment]
@fauskanger Yep, all you need to do to reference your fork and distribute it as part of your project is to reference your repo. With `pip` it's `pip install git+https://github.com/fauskanger/D3DShot#egg=D3DShot`. You can (and should) add it to your [requirements](https://pip.pypa.io/en/latest/reference/requirements-file-format/) file as well

For `poetry`, I'm not sure about the `add` command, but in the `pyproject.toml` it should look something like this according to their doc: `D3DShot= { git = "https://github.com/fauskanger/D3DShot.git", branch = "master" }`

**Comment by fauskanger** [comment]
Thanks! And more thanks for all your great work on the D3DShot package 👌

**Comment by piratajack2003** [comment]
the extracted file where should they be put?

**Comment by mathisHeidrich** [comment]
> the extracted file where should they be put?

You can put them anywhere. Once you made the changes and installed it with pip you can delete the folder.
